src/Silex/ControllerResolver.php

```diff
@@ -37,6 +37,9 @@ public function __construct(Application $app, LoggerInterface $logger = null)
         parent::__construct($logger);
     }
 
+    /**
+     * This method can be dropped once Symfony 3.0 is not supported anymore.
+     */
```

Comment:
When Symfony 3.0 is not supported anymore, that's the whole class that can be removed, right?